modules/monitoring/manifests/checks/cache.pp

```diff
@@ -4,9 +4,15 @@
 #
 # === Parameters
 #
+# [*enabled*]
+#   This variable enables to define whether to perform Icinga Elastic-Cache checks.
```

Comment:
It's probably worth specifying here that `true` is the default (according to line 17).

Reply:
Done